doc/nrf/ug_nrf5340.rst

```diff
@@ -589,6 +591,42 @@ Additionally, the memory partitions must be defined and include:
 Samples and applications built for Thingy:53 enable simultaneous update of multiple images by default.
 To learn more about Thingy:53, see :ref:`ug_thingy53`.
 
+MCUboot's serial recovery of the networking core image
```

Comment:
Two doubts about this section:

- Shouldn't this section be part of the MCUboot documentation? After all, recovering different cores can be a feature not restricted to only nRF5340 DK. In fact, we have also the guide for Thingy:53, so wouldn't this information be valid there as well?
- Shouldn't this section be a subsection of Simultaneous multi-image DFU, not a top section?

Reply:
I'm guessing this depends on CONFIG_NRF53_RECOVERY_NETWORK_CORE which should be strictly related to the 5340. @nvlsianpu ?

But yeah, I agree this should go in as a subsection

Reply:
Section is specific to nRF5340 mcuboot downstream feature. Wher I should place it then?

Shouldn't this section be a subsection of Simultaneous multi-image DFU, not a top section?
No, the simultaneous multi-image DFU is one of two way to make this feature enabled.

Reply:
^^
@FrancescoSer I put the documentation update in this file only because it already contains documentation on CONFIG_NRF53_MULTI_IMAGE_UPDAT.